plugins/search/providers/elastic_search/lib/api/enum/KalturaESearchGroupOrderByFieldName.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * @package plugins.elasticSearch
+ * @subpackage api.enum
+ */
+class KalturaESearchGroupOrderByFieldName extends KalturaStringEnum
+{
+	const UPDATED_AT = 'updated_at';
+	const CREATED_AT = 'created_at';

```

Comment:
Add 
const SCREEN_NAME = 'screen_name';
const USER_ID = 'puser_id';
to KalturaESearchUserOrderByFieldName
And inherit here.